src/components/Comment/CommentHeader.js

```diff
@@ -88,7 +88,7 @@ const titleDate = string => dateTimeFormat(new Date(string))
 
 const DefaultLink = ({ children }) => children
 
-export const CommentHeader = ({t, commentId, Link = DefaultLink, timeago, ...displayAuthor, createdAt, updatedAt, credential, highlighted}) => {
+export const CommentHeader = ({t, commentId, Link = DefaultLink, timeago, createdAt, updatedAt, credential, highlighted, ...displayAuthor}) => {
```

Comment:
While you're at it, could you also break the long line down into multiple lines?